packages/backend/src/nest/storage/userProfile/userProfile.store.ts

```diff
@@ -82,6 +130,12 @@ export class UserProfileStore extends EncryptedKeyValueStoreBase<EncryptedAndSig
     this.deferredProfiles = []
```

Comment:
Just a note but general, first-class support for "deferred" things like this that arrive out of order would be a great thing to have in a stack. Like, anything that doesn't validate gets checked again when new evidence arrives.